### ISSUE TITLE
Cypress 3.4.1 can not upload the result to Dashboard

### ISSUE THREAD

**Issue by gracesianturi** [cypress-io/cypress#4948]
### Current behavior: 

When running `cypress --run --record --key XXX` , the command prompt doesn't run any test. 

### Desired behavior: 

Cannot run the record key and Cypress 3.4.1 didn't upload the test to Dashboard Cypress

### Steps to reproduce: (app code and test code)

1. Go to project file
2. Open command prompt
3. Paste the record key from Cypress apps : `cypress run --record --key XXX`
4. The result is 
![Capture_Record_Key](https://user-images.githubusercontent.com/29389000/62674928-c9b51480-b9ce-11e9-94f2-3d8d1551cb6c.JPG)

### Version

Cypress 3.4.1
System : Windows 10
Browser : Chrome Version 75.0.3770.142

**Comment by jennifer-shehane** [comment]
This is the intended behavior as of version 3.4.1 of Cypress as noted in the [Changelog](https://on.cypress.io/changelog#3-4-1).

Because you have downloaded Cypress directly or globally, this warning displays when trying to record. 

>A warning message now displays if you attempt to run the Cypress binary directly instead of going through the cypress npm package. Addresses #1573.

This was addressing this issue: https://github.com/cypress-io/cypress/issues/1573

**TLDR:** Recording to the Dashboard never properly worked when run directly from the downloaded binary and will only correctly work when run after being installed through an npm package, so we made it impossible to run. https://github.com/cypress-io/cypress/issues/756#issuecomment-380916201

### Workaround

Install cypress as an npm package as described here in our documentation. https://on.cypress.io/installing-cypress#npm-install

**Comment by gracesianturi** [comment]
Thank you for helping me @jennifer-shehane . actually I have downloaded cypress 3.4.1 using npm. I added this script in package.json
![Capture](https://user-images.githubusercontent.com/29389000/62683341-5e783c00-b9e8-11e9-98b3-74b6c425fc01.JPG)

 When i run " npm run cy:run " , all the test will run in command prompt but can't upload the result to Dashboard. 
![Capture_Runner](https://user-images.githubusercontent.com/29389000/62683675-33dab300-b9e9-11e9-80e5-702fd0f228f3.JPG)

**Comment by jennifer-shehane** [comment]
@gracesianturi I suspect you have cypress globally installed as well as npm installed. 

In your first screenshot, you are executing `cypress run` - this will run the globally installed version if you have one, unless you have defined an alias in your bash scripts for this command specifically.

In your second comment's screenshot, you are executing `npm run cy:run` - this will always run from the npm installed version. 

### Suggestions

1. Run `npm uninstall cypress --global`
2. Add the `--record` flag to the `npm run cy:run` command to record with the npm installed version. [To pass arguments to an npm script, you need to pass `--`](https://docs.npmjs.com/cli/run-script), so you would run `npm run cy:run -- --record --key xxxx`
3. I suggest adding a new record key and using the new one since you pasted a screenshot with your record key which should remain secret. https://on.cypress.io/projects#Authentication

**Comment by gracesianturi** [comment]
Thank you for your information @jennifer-shehane . It works.
I am so happy for that 👍 
Thank you very much
